llvm/test/CodeGen/RISCV/reassoc-shl-addi-add.ll

```diff
@@ -0,0 +1,113 @@
+; RUN: llc -mtriple=riscv32-pc-unknown-gnu -mattr=+zba %s -o - | FileCheck %s
```

Comment:
```suggestion
; RUN: llc -mtriple=riscv32-unknown-elf -mattr=+zba %s -o - | FileCheck %s
```

Please also use `./llvm/utils/update_llc_test_checks.py` to get the CHECK lines, they're far easier to maintain than hand-written lines.